jfrog-cli/artifactory/utils/golang/project/project.go

```diff
@@ -69,11 +77,13 @@ func (project *goProject) PublishPackage(targetRepo, buildName, buildNumber stri
 
 	// Temp directory for the project archive.
 	// The directory will be deleted at the end.
-	err = fileutils.CreateTempDirPath()
-	if err != nil {
-		return err
+	if !fileutils.IsTempDirInit() {
```

Comment:
This can be removed, since the *fileutils.CreateTempDirPath()* can be called further upstream.

Reply:
Done